script.js

```diff
@@ -12,3 +12,350 @@ mobileLinks.addEventListener('click', () => {
+              when an unknown printer took a galley of type and scrambled it
+              1960s with the releax map lapora verita.`,
+    liveVersion: 'See live',
+    source: 'See Source',
+  },
+  {
+    title: 'Profesional Art Printing Data',
+    closeBtn: '&#x3A7',
+    paragraph: `A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard`,
+    featuredImage: {
+      src: 'assets/Snapshoot Portfolio.png',
+      alt: 'featured image',
+    },
+    technologyArticle: ['html', 'bootstrap', 'Ruby'],
+    prjBtn: 'See Project',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s with the releaLorem Ipsum is simply
+              dummy text of the printing and typesetting industry. Lorem Ipsum
+              has been the industry's standard dummy text ever since the 1500s,
+              when an unknown printer took a galley of type and scrambled it
+              1960s with the releax map lapora verita.`,
+    liveVersion: 'See live',
+    source: 'See Source',
+  },
+  {
+    title: 'Profesional Art Printing Data',
+    closeBtn: '&#x3A7',
+    paragraph: `A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard`,
+    featuredImage: {
+      src: 'assets/Snapshoot Portfolio.png',
+      alt: 'featured image',
+    },
+    technologyArticle: ['html', 'bootstrap', 'Ruby'],
+    prjBtn: 'See Project',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a galley of
+              type and scrambled it 1960s with the releaLorem Ipsum is simply
+              dummy text of the printing and typesetting industry. Lorem Ipsum
+              has been the industry's standard dummy text ever since the 1500s,
+              when an unknown printer took a galley of type and scrambled it
+              1960s with the releax map lapora verita.`,
+    liveVersion: 'See live',
+    source: 'See Source',
+  },
+  {
+    title: 'Profesional Art Printing Data',
+    closeBtn: '&#x3A7',
+    paragraph: `A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard`,
+    featuredImage: {
+      src: 'assets/Snapshoot Portfolio.png',
+      alt: 'featured image',
+    },
+    technologyArticle: ['html', 'bootstrap', 'Ruby'],
+    prjBtn: 'See Project',
+    description: `Lorem Ipsum is simply dummy text of the printing and typesetting
+              industry. Lorem Ipsum has been the industry's standard dummy text
```

Comment:
- [ optional] great job your pop works well, you can make it better by making it look more like the figma version
![Screenshot (167)_LI](https://user-images.githubusercontent.com/65298048/174440069-66fcbc69-ba95-41bf-a595-0a0f5c07b1bb.jpg)
- [ optional] since the previous reviewer forgot to mention this, please fix the up down scrolling of the mobile version so it can fit screen (full height.)
![Screenshot (168)](https://user-images.githubusercontent.com/65298048/174440173-eee29b83-0e78-46eb-ad41-ee7b4d3ab96c.png)